modules/ballerina-lang/src/main/java/org/ballerinalang/util/diagnostic/DiagnosticCode.java

```diff
@@ -62,7 +62,6 @@
     RETRY_CANNOT_BE_OUTSIDE_TRANSACTION_FAILED_BLOCK("retry.cannot.be.outside.transaction.failed.block"),
     INVALID_RETRY_COUNT("invalid.retry.count"),
     TRANSFORM_STATEMENT_INVALID_INPUT_OUTPUT("transform.statement.invalid.input.output"),
-    TRANSFORM_STATEMENT_EMPTY_BODY("transform.statement.empty.body"),
 
```

Comment:
Shall we remove the property "error.transform.statement.empty.body" from compiler.properties as well?